Sources/NIO/System.swift

```diff
@@ -108,8 +108,14 @@ private func isBlacklistedErrno(_ code: Int32) -> Bool {
 }
 
 private func assertIsNotBlacklistedErrno(err: CInt, where function: StaticString) -> Void {
+    debugOnly {
+        preconditionIsNotBlacklistedErrno(err: err, where: function)
+    }
+}
```

Comment:
Sadly I think this `debugOnly` block is causing issues with the performance tester. Can you replace this with `assert(!isBlacklistedErrno(err), "blacklisted errno \(err) \(String(cString: strerror(err)!)) in \(function))")`

Reply:
The `debugOnly` block has been reverted in the latest comit